indico/modules/events/contributions/templates/forms/contribution_person_widget.html

```diff
@@ -82,8 +80,8 @@
 
             function renderPerson(person) {
                 var $personRow = $('<div>').addClass('person-row');
-                var $personName = $('<span>').text(person.name);
-                var $personStatus = $('<span>').addClass('person-status');
+                var $personName = $('<div>').addClass('name').text(person.name);
+                var $personStatus = $('<div>').addClass('person-status');
                 var $personRoles = $('<span>').addClass('person-roles');
                 var $personButtons = $('<span>').addClass('person-buttons');
                 var $buttonRemove = $('<a>').addClass('i-button-icon danger icon-close').attr('title', $T.gettext("Remove person"));
```

Comment:
Ah.. I was testing it in the console but forgot that `{...}` on its own does not create an object without being inside an expression. Apparently quoting it is just needed for pre-ES5 browsers so we don't need the quotes: http://caniuse.com/#feat=es5